src/scss/components/_ratings.scss

```diff
@@ -1,30 +1,37 @@
-$star-count: 6;
-$star-size-small: 16px;
-$star-size-medium: 28px;
-$star-size-large: 42px;
-$star-spacing: $star-count;
+$star-count             : 6;
+$star-size-extraSmall   : 12px;
+$star-size-small        : 16px;
```

Comment:
Might be best to change `$star-size-small` to `$star-size-default` as that's the default sizing used.

Then the new size can be `$star-size-small` as it's making it smaller than the default (make sense?).

Also, would be good to change the names to be `$star-size--small` (double dash as they're modifiers) if that doesn't mean too much re-work.